bootstrap/bootstrap-kubestellar.sh

```diff
@@ -76,7 +76,7 @@ kcp_version() {
 }
 
 kcp_get_latest_version() {
-    curl -sL https://github.com/kcp-dev/kcp/releases/latest | grep "</h1>" | head -n 1 | sed -e 's/<[^>]*>//g' | xargs
+    curl -sL https://github.com/kcp-dev/kcp/releases/latest | grep "</h1>" | tail -n 1 | sed -e 's/<[^>]*>//g' | xargs
```

Comment:
This is not wrong, but I wonder if you would rather have a more robust search based on more details of the content (e.g., `fgrep '<h1 data-view-component="true"'`).  Following is what I see right now.

```console
(base) mspreitz@mjs12 kubestellar % curl -sL https://github.com/kcp-dev/kcp/releases/latest | grep "</h1>"
      <h1 id="search-suggestions-dialog-header" class="sr-only">Search code, repositories, users, issues, pull requests...</h1>
      </h1>
      </h1>
        <h1 data-view-component="true" class="d-inline mr-3">v0.20.0</h1>
```

Reply:
@MikeSpreitzer ideally, I would like to have a file `latest.txt` and `stable.txt` in the repo pointing to the stable and latest release.

See what go does: `curl 'https://go.dev/VERSION?m=text'`

Reply:
OTOH, including in a commit a file that references the commit is problematic.  That is why I think that a GitHub API would be the best solution.

Reply:
If we had a different repo in which we could put the pointer to "stable" and "latest", that would be a resolution.

Reply:
How do the go developer control what `curl 'https://go.dev/VERSION?m=text'` produces?

Reply:
Also, one of the answers in https://stackoverflow.com/questions/51933636/url-for-latest-stable-go-release says that `https://go.dev/dl/?mode=json` should be read instead.

Reply:
> OTOH, including in a commit a file that references that commit (or a tag that will be created after the commit exists) is problematic. That is why I think that a GitHub API would be the best solution.

@MikeSpreitzer I don't share your concern. The main branch will hold the stable/latest files, not the tag releases. The stable could be rolled back anytime if  an issue is discovered.
After the binary release is done, the main branch will decide to update the files as appropriate to advance the users.
The url to stable/latest.txt should be immutable, so it should be in the main branch.